dataland-backend/src/main/kotlin/org/dataland/datalandbackend/model/sme/SmeData.kt

```diff
@@ -0,0 +1,86 @@
+package org.dataland.datalandbackend.model.sme
+
+import org.dataland.datalandbackend.annotations.DataType
+import org.dataland.datalandbackend.model.enums.commons.YesNoNa
+import org.dataland.datalandbackend.model.enums.sme.CompanyAgeBracket
+import org.dataland.datalandbackend.model.enums.sme.EnergyEfficiencyBracket
+import org.dataland.datalandbackend.model.enums.sme.EnergyProductionBracket
+import org.dataland.datalandbackend.model.enums.sme.HeatSource
+import org.dataland.datalandbackend.model.enums.sme.Industry
+import java.math.BigDecimal
+
+/**
+ * --- API model ---
+ * Fields of the questionnaire for the SME framework
+ */
+@DataType("sme")
+data class SmeData(
+    val industry: Industry?,
+
+    val financialYear: Long?,
+
+    val totalRevenue: BigDecimal?,
+
+    val researchAndDevelopmentExpenses: BigDecimal?,
+
+    val energyEfficiencyInvestments: EnergyEfficiencyBracket?,
+
+    val operatingCosts: BigDecimal?,
+
+    val totalAssets: BigDecimal?,
+
+    val yearsSinceFounded: CompanyAgeBracket?,
+
+    val productCategoryHighestSales: String?,
```

Comment:
prodcutCategoryWithHighestSales? Is more readible to me at least. Same for the following four.

Reply:
Ja stimmt. Habe ich geändert das Excel passe ich auch an.